src/vs/workbench/contrib/welcome/page/browser/welcomePage.contribution.ts

```diff
@@ -35,13 +37,33 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 		}
 	});
 
-Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench)
-	.registerWorkbenchContribution(WelcomePageContribution, LifecyclePhase.Restored);
+class WelcomeContributions {
+	constructor(
+		@IConfigurationService configurationService: IConfigurationService,
+	) {
+		const previewFeaturesEnabled: boolean = configurationService.getValue('workbench')['enablePreviewFeatures'];
```

Comment:
This will only be run once at startup? Don't we want to listen to this being changed?

Reply:
no, the design is to only run at startup.  we discussed in shiproom channel and previous comments.